io.catenax.generic.digital_product_passport/2.0.0/DigitalProductPassport.ttl

```diff
@@ -0,0 +1,1142 @@
+   samm:description "The actual value of the concentration as float."@en ;
+   samm:dataType xsd:float .
+
+:FootprintList a samm-c:List ;
+   samm:preferredName "Footprint List"@en ;
+   samm:description "Footprint List for the carbon and environmental footprint."@en ;
+   samm:dataType :FootprintEntity .
+
+:LifeEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Life Enumeration"@en ;
+   samm:description "Enumeration with the possible units day, month, cycle, year and runningOrOperatingHour."@en ;
+   samm:see <https://eclipse-esmf.github.io/samm-specification/snapshot/appendix/unitcatalog.html> ;
+   samm:dataType samm:curie ;
+   samm-c:values ( "unit:day"^^samm:curie "unit:month"^^samm:curie "unit:year"^^samm:curie "unit:cycle"^^samm:curie "unit:runningOrOperatingHour"^^samm:curie ) .
+
+:LifeValueCharacteristic a samm-c:Quantifiable ;
+   samm:preferredName "Life Value Characteristic"@en ;
+   samm:description "Characteristic for the life span value as an integer."@en ;
+   samm:dataType xsd:integer .
+
+:LifeTypeEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Life Type Enumeration"@en ;
+   samm:description "Enumeration with the values guaranteed lifetime, technical lifetime and mean time between failures."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "guaranteed lifetime" "technical lifetime" "mean time between failures" ) .
+
+:LinearCharacteristic a samm:Characteristic ;
+   samm:preferredName "Linear Characteristic"@en ;
+   samm:description "Characteristic for linear measurements of an item."@en ;
+   samm:dataType :LinearEntity .
+
+:MassCharacteristic a samm:Characteristic ;
+   samm:preferredName "Mass Characteristic"@en ;
+   samm:description "Characteristic for mass measurements of an item."@en ;
+   samm:dataType :MassEntity .
+
+:CapacityCharacteristic a samm:Characteristic ;
+   samm:preferredName "Capacity Characteristic"@en ;
+   samm:description "Characteristic for capacity measurements of an item."@en ;
+   samm:dataType :CapacityEntity .
+
+:PhysicalStateEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Physical State Enumeration"@en ;
+   samm:description "Enumeration with the values solid, liquid and gas."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "solid" "liquid" "gas" ) .
+
+:EoriTrait a samm-c:Trait ;
+   samm-c:baseCharacteristic :IdentifierCharacteristic ;
+   samm-c:constraint :EoriConstraint .
+
+:CodeEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Code Enumeration"@en ;
+   samm:description "Enumeration for GTIN, hash, DID, ISBN, TARIC or other."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "TARIC" "GTIN" "DID" "ISBN" "hash" "other" ) .
+
+:IdentifierEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Identifier Enumeration"@en ;
+   samm:description "Enumeration with \"PartInstanceID\" \"BatchID\" and \"ModelID\"."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "PartInstanceID" "BatchID" "ModelID" ) .
```

Comment:
Wasn't there a decision on architectural level to write ID in Camelcase as Id? So probably you need to adjust this here, too.